File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/funwithtests/TestClockComponent.java
package com.przemyslawlusnia.vocabularycreator.funwithtests;

import com.przemyslawlusnia.vocabularycreator.core.di.AppScope;
import dagger.Component;

@AppScope
@Component(modules = {TestClockModule.class})
public interface TestClockComponent extends ClockComponent {

}

File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/funwithtests/TestClockComponent.kt
package com.przemyslawlusnia.vocabularycreator.funwithtests

import com.przemyslawlusnia.vocabularycreator.core.di.AppScope
import dagger.Component

@AppScope
@Component(modules = arrayOf(TestClockModule::class))
interface TestClockComponent : ClockComponent
